### DISCUSSION TITLE
has this been abandoned?

### DISCUSSION THREAD

**Discussion by devmuppet** [jef/streetmerchant#2996] [Help]
not seeing updates to multiple shops/series(3070ti) and containers?

tried to update some code to add new shop urls and the 3070ti + 3080 12gb models but the code doesnt work... tried running the container and that also seems well out of date.

Is this now abandoned?

**Comment by Mr-Tech-13** [comment]
No, There are still updates, just not as regular. What errors do you get?

**Comment by jef** [comment]
i have lost passion into the project, but i am still updating links, accepting pull requests, and updating dependencies.

**Reply by Mr-Tech-13** [reply]
Don't get too burned out, great work!

**Reply by devmuppet** [reply]
Understandably dude, be proud... Without it there would be a lot of sad people in the world. Much love for this!

**Reply by jef** [reply]
Thank you @Mr-Tech-13 and @devmuppet :)
